src/frontend/src/components/common/Navbar/index.tsx

```diff
@@ -10,22 +10,26 @@ function NavBar({
   return (
     <nav className={CSSclass}>
       <ul className="navbar">
-        {pages.map((page, key) => {
-          return page.enable === true ? (
-            <li key={key}>
-              <NavLink
-                className={`bcgovbtn bcgovbtn__tertiary bcgovbtn__tertiary--dark ${
-                  page.link === currentPath ? 'active' : ''
-                }`}
-                to={page.link}
-              >
-                {page.label}
-              </NavLink>
-            </li>
-          ) : (
-            ''
-          );
-        })}
+        {pages
+          .filter((page) => page.enable)
+          .map((page) => {
+            return (
+              <li key={page.id}>
+                {page.isDivider ? (
+                  <hr />
```

Comment:
personally, not sure this looks right. What if we want a horizontal bar. I would rather have a class-name defined in our interface. 
The class-name should define the bottom border padding etc.

Reply:
Good point and it got missed. Created a sub-issue for this. will fix in the next PR. 
Just want everyone to leverage this template as quickly as possible.

[UTOPIA-836](https://apps.itsm.gov.bc.ca/jira/browse/UTOPIA-836)